mmdet3d/models/dense_heads/vote_head.py

```diff
@@ -119,9 +119,19 @@ def _extract_input(self, feat_dict):
             torch.Tensor: Features of input points.
             torch.Tensor: Indices of input points.
         """
-        seed_points = feat_dict['seed_points']
-        seed_features = feat_dict['seed_features']
-        seed_indices = feat_dict['seed_indices']
+
+        # for imvotenet
+        if 'seed_points' in feat_dict.keys() and \
```

Comment:
`'seed_points' in feat_dict`, may omit `.key()`

Reply:
refer to https://github.com/open-mmlab/mmdetection3d/pull/297#discussion_r586313357